applications/incompressible_fluid_application/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme_dpg_enriched.h

```diff
@@ -69,7 +69,12 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #include "containers/array_1d.h"
 #include "utilities/openmp_utils.h"
 #include "utilities/coordinate_transformation_utilities.h"
-#include "custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme.h"
+/* NOTE: the include below uses an ugly relative path because this file is also used within the trilinos application.
+ * The correct way to do this would be to either formalize the dependence (so that the trilinos application explicitly
+ * includes this folder in its include path) or invert it (the incompressible fluid depending on the trilinos application),
+ * which is probably the better option for Kratos. Until either of those happen, we need this patch. JC
+ */
+#include "../incompressible_fluid_application/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme.h"
```

Comment:
do we actually need this file? can't we remove it for good?

Reply:
I do not need it, and looks like you also don't. @jrubiogonzalez, @ddiezrod do you use this file?